packages/@aws-cdk/cloudformation-include/test/yaml-templates.test.ts

```diff
@@ -0,0 +1,382 @@
+import '@aws-cdk/assert/jest';
+import * as core from '@aws-cdk/core';
+import * as path from 'path';
+import * as inc from '../lib';
+import * as futils from '../lib/file-utils';
+
+// tslint:disable:object-literal-key-quotes
+/* eslint-disable quotes */
+
+describe('CDK Include', () => {
+  let stack: core.Stack;
+
+  beforeEach(() => {
+    stack = new core.Stack();
+  });
+
+  test('can ingest a template with all long cfn functions except Fn::Sub, Fn::GetAtt, and Fn::Join, and output it unchanged', () => {
+    includeTestTemplate(stack, 'long-form-functions.yaml');
+
+    expect(stack).toMatchTemplate(
+      loadTestFileToJsObject('long-form-functions.yaml'),
+    );
+  });
+
+  test('can ingest a template with long form Fn::Join and Fn::GetAtt, and output it unchanged', () => {
+    includeTestTemplate(stack, 'long-form-sample.yaml');
+
+    expect(stack).toMatchTemplate(
+      loadTestFileToJsObject('long-form-sample.yaml'),
+    );
+  });
```

Comment:
Let's have just one test here: `can ingest a YAML template with long-form CloudFormation functions and output it unchanged`.